605.can-place-flowers.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/can-place-flowers/
+ */
+
+/**
+ * @param {number[]} flowerbed
+ * @param {number} n
+ * @return {boolean}
+ */
+var canPlaceFlowers = function (flowerbed, n) {
+	let counter = 0;
+
+	for (let i = 0; i < flowerbed.length; i++) {
+		if (flowerbed[i] === 0) {
+			if (flowerbed[i - 1] !== 1 && flowerbed[i + 1] !== 1) {
+				flowerbed[i] = 1;
+				counter++;
+			}
+		}
```

Comment:
```suggestion
		if (flowerbed[i] === 1) {
                    continue;
                }
        
                if (flowerbed[i - 1] !== 1 && flowerbed[i + 1] !== 1) {
                    flowerbed[i] = 1;
                    counter++;
                }
```
You can also make it less nesting by doing this